### PR TITLE
[DOC-39] Ed-Fi API-Guidelines Space migration

### PR DESCRIPTION
**Pull request by chrisgocode**

### CODE REVIEW

File: docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/api-design-guidelines/resources/readme.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  },
+  ...
+}
+```
+
+## Resource Extensions
+
+The Ed-Fi UDM can be extended to augment the structure of existing entities or
+create entirely new entities (cf [Ed-Fi Data Standard Extension
+Framework](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/56655917/Ed-Fi+Data+Standard+Extension+Framework)).
```
Comment (stephenfuqua) [inline docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/api-design-guidelines/resources/readme.md:138]:
```suggestion
Framework](../../../../extension-framework.md).
```

File: docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Unless otherwise stated, the guidelines provided by this document apply to both
+_compatible_ and _aligned_ API applications.
+
+## Ed-Fi API Specifications
+
+### Resource API
+
+The [Ed-Fi Unifying Data
+Model](https://edfi.atlassian.net/wiki/spaces/EFDS/pages/129990704/Ed-Fi+Unifying+Data+Model)(UDM)
```
Comment (stephenfuqua) [inline docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md:26]:
```suggestion
The [Ed-Fi Unifying Data Model](../../data-standard/udm.md)(UDM)
```

**Review by stephenfuqua — APPROVED** [review]

**Comment by stephenfuqua** [comment]
Needs to be approved by someone other than me. From my perspective, this is ready to go.

File: docs/reference/1-data-exchange/api-guidelines/summary-of-changes-v40.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+* API Implementation Guidelines
+
+  * Handling Authentication and Authorization
+
+  * Handling Non-Repudiation
+
+  * Handling Optimistic Concurrency with ETags
+
+  * Handling Web Cache Validation with ETags
```
Comment (roberthunterjr) [inline docs/reference/1-data-exchange/api-guidelines/summary-of-changes-v40.md:66]: This might be a good use case for https://ascii-tree-generator.com/

**Review by roberthunterjr — APPROVED** [review]

File: docs/reference/1-data-exchange/api-guidelines/summary-of-changes-v40.md (hunk 3 above)
Comment (stephenfuqua) [reply docs/reference/1-data-exchange/api-guidelines/summary-of-changes-v40.md:66]: That might work... though I tend to use it only for filesystems, not table of contents (though they're almost the same thing). I don't like all of the extra whitespace between bullet points though, so I'm going to fix that.

File: docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Ed-Fi API software _must_ utilize the OAuth 2.0 API specification for managing authentication, though the implementation _may_ use non-Ed-Fi software to provision OAuth-related services.
+
+:::tip
+
+The Ed-Fi ODS/API Platform exclusively uses a built-in OAuth 2.0 provider, which cannot be replaced by an off-the-shelf component. This need not be true of other implementations.
+
+:::
+
+Other specifications managed by the Ed-Fi Alliance include:
```
Comment (ecleamus) [inline docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md:51]: Are we removing all instances of extra spacing in bullets? @stephenfuqua

**Review by ecleamus — COMMENTED** [review]
Other than my question about spaces, looks good.

File: docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md (hunk 4 above)
Comment (stephenfuqua) [reply docs/reference/1-data-exchange/api-guidelines/design-and-implementation-guidelines/scope.md:51]: Not a strict requirement. I personally prefer not to have extra whitespace around each bulleted item, but I haven't been pushing the issue. This specific list seemed a little more egregious than others.

**Review by ecleamus — APPROVED** [review]

**Pull request merged**